<?xml version="1.0"?>
<launch>

    <rosparam file="$(find search_service)/config/ahg.yaml"/>
    <arg name="agent1_pose_topic" default="global_pose_a1_121"/>
    <arg name="agent2_pose_topic" default="hsr/global_pose"/>
    <arg name="agent3_pose_topic" default="a1_950/global_pose"/>
    <arg name="agent1_map_topic" default="costmap_node/costmap/costmap"/>
    <arg name="agent2_map_topic" default="hsr/costmap"/>
    <arg name="agent3_map_topic" default="a1_950/costmap"/>
    <arg name="max_x" default="16.0"/>
    <arg name="min_x" default="-15.0"/>
    <arg name="max_y" default="20.0"/>
    <arg name="min_y" default="-30"/>
    <arg name="num_agents" default="3"/>

    <!--<node name="single_serach_server" pkg="search_service" type="singlesearch_server">-->
    <node name="multiserach_action" pkg="search_service" type="mtsp_server" output="screen">
      <param name="MAX_X" value="$(arg max_x)"/>
      <param name="MAX_y" value="$(arg max_y)"/>
      <param name="MIN_X" value="$(arg min_x)"/>
      <param name="MIN_Y" value="$(arg min_y)"/>
      <param name="AGETNT1_POSE_TOPIC" value="$(arg agent1_pose_topic)"/>
      <param name="AGETNT2_POSE_TOPIC" value="$(arg agent2_pose_topic)"/>
      <param name="AGETNT3_POSE_TOPIC" value="$(arg agent3_pose_topic)"/>
      <param name="AGENT1_MAP_TOPIC" value="$(arg agent1_map_topic)"/>
      <param name="AGENT2_MAP_TOPIC" value="$(arg agent2_map_topic)"/>
      <param name="AGENT3_MAP_TOPIC" value="$(arg agent3_map_topic)"/>
      <param name="NUM_AGENT" value="$(arg num_agents)"/>
  </node>



</launch>
